config/routes.rb

```diff
@@ -33,8 +33,10 @@
   end
 
   scope 'admin' do
+    require 'sidekiq/web'
     mount BadgesEngine::Engine , at: '/'
     mount PresentationsEngine::Engine, at: '/'
+    mount Sidekiq::Web => '/sidekiq'
```

Comment:
do you need the sidekiq web on prod? you could check for env and load it only for dev ...